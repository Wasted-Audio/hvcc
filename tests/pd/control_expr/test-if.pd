#N canvas 1245 634 450 300 12;
#X obj 52 25 loadbang;
#X obj 52 160 print;
#X msg 106 85 3.1;
#X msg 52 85 2.3;
#X obj 52 50 t b b b;
#X obj 52 126 expr if($f1 < $f2 \, $f1 \, 0);
#X connect 0 0 4 0;
#X connect 2 0 5 1;
#X connect 3 0 5 0;
#X connect 4 0 3 0;
#X connect 4 1 2 0;
#X connect 5 0 1 0;
