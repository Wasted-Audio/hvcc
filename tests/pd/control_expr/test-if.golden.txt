[@ 0.000] print: 2.3
